### PR TITLE
[ko] Update translation

### PR DESCRIPTION
**Pull request by marocchino**
- Update metaprogramming
- Reduce some verbose sentences
- Omit trailing and
- Unify words: macro

### CODE REVIEW

**Comment by marocchino** [comment]
@Dalgona, @malkoG, @devleoper, @riseshia review please.

File: ko/lessons/advanced/otp-concurrency.md [hunk 1]
```diff
@@ -39,20 +38,18 @@ defmodule SimpleQueue do
...
-현재 큐의 상태를 보여주는 코드와 값을 제거하는 코드를 포함시켜, 동기 응답이 어떻게 동작하는지 보죠.
+패턴매칭을 이용하여, 다양한 요청과 상태에 따라 콜백을 정의할 수 있습니다. [`GenServer.handle_call/3`](http://elixir-lang.org/docs/v1.1/elixir/GenServer.html#c:handle_call/3) 문서에서 허용되는 반환 값의 전체 목록을 확인할 수 있습니다.
 
+현재 큐의 상태를 보여주는 기능과 값을 제거하는 기능을 추가해, 동기 응답이 어떻게 동작하는지 보죠.
 
 ```elixir
 defmodule SimpleQueue do
   use GenServer
-  
+
```
Comment (devleoper) [inline ko/lessons/advanced/otp-concurrency.md]: :100:

File: ko/lessons/advanced/umbrella-projects.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ for each application in the apps/ directory.
 
 셸 명령어를 입력했을 때 결과에서 보신 것처럼, Mix가 디렉터리 두 개로 된 프로젝트의 작은 뼈대를 만들었습니다.
 
-  - `apps/` - 서프 프로젝트(자식 프로젝트)가 위치해 있는 곳
+  - `apps/` - 하위(자식) 프로젝트가 위치하는 곳
```
Comment (malkoG) [inline ko/lessons/advanced/umbrella-projects.md]: 👍

File: ko/lessons/advanced/otp-supervisors.md [hunk 3]
```diff
@@ -6,15 +6,15 @@ order: 5
...
 
-슈퍼바이저(Supervisor)는 다른 프로세스를 감시한다는, 단 하나의 목적에 특화된 프로세스입니다. 자식 프로세스가 실패하면 자동으로 재시작해주므로 장애에 대한 내성이 높은(Fault-tolerant) 애플리케이션을 만들 수 있게 해줍니다.
+슈퍼바이저(Supervisor)는 다른 프로세스의 감시라는 단 하나의 목적에 특화된 프로세스입니다. 자식 프로세스가 실패하면 자동으로 재시작해주는 것으로 장애에 대한 내성이 높은(Fault-tolerant) 애플리케이션을 만들 수 있게 해줍니다.
 
 {% include toc.html %}
 
 ## 설정
 
-슈퍼바이저의 마법은 `Supervisor.start_link` 함수의 내부에 있습니다. 슈퍼바이저 프로세스를 실행하면서, 슈퍼바이저가 자식을 관리하기 위해 사용할 전략을 정의할 수 있습니다.
+슈퍼바이저의 마법은 `Supervisor.start_link` 함수의 내부에 있습니다. 슈퍼바이저와 그자식 프로세스를 실행하면서, 슈퍼바이저가 자식 프로세스를 관리하기 위해 사용할 전략을 정의할 수 있습니다.
```
Comment (malkoG) [inline ko/lessons/advanced/otp-supervisors.md]: how about `슈퍼바이저와 그자식` -> `슈퍼바이저와 그(or 슈퍼바이저)의 자식`?
it seems like slang 😢

Comment (devleoper) [inline ko/lessons/advanced/otp-supervisors.md]: just **슈퍼바이저와 자식 프로세스를 실행하면서** sounds not too bad.

Comment (devleoper) [inline ko/lessons/advanced/otp-supervisors.md]: @malkoG Your suggestion seems to be applied. Do you like these changes? If so, Can you make your review as **approved**? 😀

**Comment by devleoper** [comment]
Merged! Thanks for your continued support 😄 

_For referencing purpose:_ Debugging section of Metaprogramming lesson is translated from #489.

**Pull request merged**